datafusion/core/Cargo.toml

```diff
@@ -39,8 +39,8 @@ avro = ["apache-avro", "num-traits", "datafusion-common/avro"]
 backtrace = ["datafusion-common/backtrace"]
 compression = ["xz2", "bzip2", "flate2", "zstd", "async-compression"]
 crypto_expressions = ["datafusion-physical-expr/crypto_expressions", "datafusion-optimizer/crypto_expressions"]
-default = ["crypto_expressions", "encoding__expressions", "regex_expressions", "unicode_expressions", "compression"]
```

Comment:
Drive by fix: I believe the double `_` `_` is a typo that results in this feature always being on. This makes it so users can disable it if needed